rocks-bin/src/install.rs

```diff
@@ -0,0 +1,15 @@
+use eyre::Result;
+use rocks_lib::config::Config;
+
+#[derive(clap::Args)]
+pub struct Install {
+    /// Name of the rock to install.
+    name: String,
+    /// Rocks version to install.
+    version: Option<String>,
+}
```

Comment:
suggestion: I'm a fan of [parse, don't validate](https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/).

Can we apply this here, by making the version a `semver::Version` and perhaps by creating a wrapper type for rock names that allow only valid characters?

It would require more involved arg parsing, but it results in nicer errors if users pass in invalid arguments.

Maybe this is for another PR, as it would mean refactoring the `Download` command, too.